python/hail/docs/doctest_setup.py

```diff
@@ -10,66 +12,28 @@
     if os.path.isdir(f):
         shutil.rmtree(f)
 
-init(log="output/hail.log", quiet=True)
-
-ds = methods.import_vcf('data/sample.vcf.bgz')
+ds = hl.import_vcf('data/sample.vcf.bgz')
 ds = ds.sample_rows(0.03)
-ds = ds.annotate_rows(useInKinship = functions.rand_bool(0.9), panel_maf = 0.1, anno1 = 5, anno2 = 0, consequence = "LOF", gene = "A", score = 5.0)
-ds = ds.annotate_rows(aIndex = 1)
-ds = methods.sample_qc(methods.variant_qc(ds))
-ds = ds.annotate_cols(isCase = True,
-                      pheno = Struct(isCase = functions.rand_bool(0.5),
-                                     isFemale = functions.rand_bool(0.5),
-                                     age = functions.rand_norm(65, 10),
-                                     height = functions.rand_norm(70, 10),
-                                     bloodPressure = functions.rand_norm(120, 20),
-                                     cohortName = "cohort1"),
-                      cov = Struct(PC1 = functions.rand_norm(0, 1)), cov1 = functions.rand_norm(0, 1), cov2 = functions.rand_norm(0, 1))
-ds.write('data/example.vds', overwrite = True)
-
-ds = methods.import_vcf('data/sample.vcf.bgz')
-ds = ds.sample_rows(0.015)
-ds = ds.annotate_rows(anno1 = 5, toKeep1 = True, toKeep2 = False, toKeep3 = True)
-ds = methods.split_multi_hts(ds)
-ds.write('data/example2.vds', overwrite=True)
-
-ds = methods.import_vcf('data/sample.vcf.bgz')
-ds.write('data/example2.multi.generic.vds', overwrite=True)
-
-ds = methods.import_vcf('data/sample.vcf.bgz')
-ds = methods.split_multi_hts(ds)
```

Comment:
did we not need all of these imports?

Reply:
I moved the burden + lmmreg ones to the places they're used (once each). This sped up doctests by about 3x.